libraries/libfc/include/fc/crypto/bls_signature.hpp

```diff
@@ -8,83 +8,49 @@
 #include <fc/reflect/variant.hpp>
 #include <bls12-381/bls12-381.hpp>
 
-
-namespace fc { namespace crypto { namespace blslib {
-
-   using namespace std;
+namespace fc::crypto::blslib {
 
    namespace config {
-      constexpr const char* bls_signature_base_prefix = "SIG";
-      constexpr const char* bls_signature_prefix = "BLS";
+      constexpr std::string_view bls_signature_prefix = "SIG_BLS_";
    };
-
+   
    class bls_signature

```

Comment:
Move constructors together and make the non-default ones `explicit`. Remove unnecessary friend declarations. Remove extra spaces. class can be simplified to:

```
   struct bls_signature
   {
         bls_signature() = default;
         bls_signature(bls_signature&&) = default;
         bls_signature(const bls_signature&) = default;
         explicit bls_signature(const bls12_381::g2& sig) : _sig(sig) {}
         explicit bls_signature(const std::string& base58str);

         bls_signature& operator=(const bls_signature& ) = default;
         std::string to_string(const yield_function_t& yield = yield_function_t()) const;
         friend bool operator==(const bls_signature& p1, const bls_signature& p2);

         bls12_381::g2 _sig;
   };
```

Reply:
An update, I just merged the PR which updates leap/main to use the latest bls12 submodule (which adds the comparison operators), so if you update this catchup from main and update this submodule, the above should now build.